tauri-app/src-tauri/src/commands/fork.rs

```diff
@@ -0,0 +1,191 @@
+use alloy_dyn_abi::JsonAbiExt;
+use alloy_json_abi::Error;
+use forker::*;
+use once_cell::sync::Lazy;
+use reqwest::Client;
+use revm::primitives::Bytes;
+use std::{collections::HashMap, sync::Mutex};
+
+/// static hashmap of fork evm instances, used for caching instances between runs
+pub static FORKS: Lazy<Mutex<HashMap<String, ForkedEvm>>> =
+    Lazy::new(|| Mutex::new(HashMap::new()));
+
+/// hashmap of cached error selectors    
+pub static SELECTORS: Lazy<Mutex<HashMap<Vec<u8>, Error>>> =
+    Lazy::new(|| Mutex::new(HashMap::new()));
+
+#[tauri::command]
+pub async fn fork_call(
+    fork_url: &str,
+    fork_block_number: u64,
+    from_address: &[u8],
```

Comment:
can these be address or `[u8; 20]` or something? maybe even something that implements https://docs.rs/serde_bytes/latest/serde_bytes/ ?

Reply:
I used &[u8] to make it easier for using the underlying function, it will error if the u8 is not valid address length